recipe/meta.yaml

```
@@ -9,8 +9,8 @@ package:
 
 source:
 
-  url: https://github.com/nasa/NASAaccess/archive/V3.4.3.tar.gz
-  sha256: 56a4c30935d08aad56c28fd23874ae9018528179eccd82528a305a1bf7ebbfe3
+  url: https://github.com/nasa/NASAaccess/archive/v4.0.0.tar.gz
```

Would make sense to use the `version` variable:

```suggestion
  url: https://github.com/nasa/NASAaccess/archive/v{{ version }}.tar.gz
```